index.html

```diff
@@ -262,53 +270,54 @@ <h2>LET'S CONNECT</h2>
             </ul>
           </div>
         </div>
-      </section>
-      <!-- Contact Form here -->
 
-      <section class="contactt">
-        <div class="ttry">
-          <form
-            class="forms"
-            action="https://formspree.io/f/moqroego"
-            method="POST"
-          >
-            <h2 class="contactme-head">Contact me</h2>
-            <p class="contactme-body">
-              If you have an application you are interested in developing, a
-              feature that you need built or a project that needs coding. I’d
-              love to help with it
-            </p>
-            <input
-              class="form-name"
-              type="text"
-              maxlength="30"
-              name="name"
-              placeholder="Full Name"
-              required
-            />
-            <input
-              class="form-email"
-              type="email"
-              name="email"
-              placeholder="Email"
-              required
-            />
-            <textarea
-              class="form-message"
-              name="message"
-              maxlength="500"
-              placeholder="Write your message here"
-              required
-            ></textarea>
-            <br />
-            <p class="error"></p>
-            <br />
-            <button class="submission" type="submit">Get in touch</button>
-          </form>
-        </div>
-        <div class="shapes">
-          <img src="images/shape.png" alt="lines" />
-        </div>
+        <!-- Contact Form here -->
+
+        <section class="contactt">
+          <div class="ttry">
+            <form
+              class="forms"
+              action="https://formspree.io/f/moqroego"
+              method="POST"
+            >
+              <h2 class="contactme-head">Contact me</h2>
+              <p class="contactme-body">
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I’d
+                love to help with it
+              </p>
```

Comment:
- [ ] After around 1100px, your contact section stops taking up the entire space and has white space around it as you can see below
![image](https://user-images.githubusercontent.com/90556221/178105829-f216964e-e8ce-4795-980d-5ce0694df97e.png)
Kindly ensure the contact section always takes up the entire width of the viewport

- [ ] The above change also needs to be implemented in the about section as it stops taking up the provided space and is just centralized on certain screen sizes above 1100px (Still desktop version)
![image](https://user-images.githubusercontent.com/90556221/178105928-82d63abe-e78f-4568-8935-b4f517efa6e3.png)